frontend/src/advising/GenericAdvisingTemplate.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Tabs, Tab } from "@material-ui/core";
+import { withRouter, RouteComponentProps } from "react-router-dom";
+
+const HomeText = styled.a`
+  font-weight: bold;
+  font-size: 36px;
+  text-decoration: none;
+  color: black;
+`;
+const StyledTab = styled(Tab)`
+  color: white !important;
+`;
+
+const Container = styled.div`
+  margin: 30px;
+  background-color: "#ff76ff";
+`;
+const TabsWrapper = styled.div`
+  margin: 20px -30px;
+`;
+
+const PATH_0 = `/advisor/notifications`;
+const PATH_1 = `/advisor/manageStudents`;
+const PATH_2 = `/advisor/templates`;
```

Comment:
Maybe we could store these constants in an array indexed by tab order!

This way to get the start tab number you could do:
```
const startTab = PATHS.findIndex(curPath); // and possibly handle the -1 case with a default tab
```

You can also then reuse the constants within `handleChange`:
```
const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
    history.push(PATHS[newValue]);
    setCurrentTab(newValue);
}
```